include/kernels/dslash_pack.cuh

```diff
@@ -47,9 +52,29 @@ namespace quda
     int_fastdiv swizzle;
     int sites_per_block;
 
+    char *packBuffer[4 * QUDA_MAX_DIM];
+    int neighbor_ranks[2 * QUDA_MAX_DIM];
+    int bytes[2 * QUDA_MAX_DIM];
+    // shmem bitfield encodes
+    // 0 - no shmem
+    // 1 - pack P2P
+    // 2 - pack IB
+    // 3 - pack P2P + IB
+    // 8 - barrier part I (just the put part)
+    // 16 - barrier part II (wait on shmem to complete, all directions) -- not implemented
```

Comment:
`16` seems to be noted as implemented in `lib/dslash_policy.cuh`---are these different bitfields? Or is this wrong/I'm misunderstanding the comment in that file?

Reply:
thanks. The comment was outdated.